apps/webapp/src/components/shared/input-token.tsx

```diff
@@ -57,31 +48,12 @@ export default function InputToken({
   inputClassName,
   setSelection,
   balances,
-  fee,
   ...props
 }: InputTokenProps) {
-  const vResult = validationResult(value, validations);
-
   const currentBalanceValueView = getCurrentBalanceValueView(selection?.asset);
-  const feeAsString = getFeeAsString(fee);
 
   return (
-    <div
-      className={cn(
-        'bg-background px-4 pt-3 pb-5 rounded-lg border flex flex-col',
-        vResult?.type === 'error' && 'border-red-400',
-        vResult?.type === 'warn' && 'border-yellow-300',
-        className,
-      )}
-    >
-      <div className='mb-2 flex items-center justify-between gap-1 md:gap-2'>
-        <p className='text-sm font-bold md:text-base'>{label}</p>
-        {vResult ? (
-          <div className={cn('italic text-[12px] md:text-[15px]', 'text-red-400')}>
-            {vResult.issue}
-          </div>
-        ) : null}
-      </div>
```

Comment:
All of this was duplicating markup from `InputBlock`, albeit with slightly different font sizes etc. So I just refactored it to use `InputBlock` directly.